deploy.sh

```diff
@@ -4,6 +4,16 @@ set -e
 if [[ "$TRAVIS_BRANCH" != "master" || "$TRAVIS_PULL_REQUEST" != "false" ]]; then
     echo "Skipping deploy for a pull request"
 else
+    echo "Importing content from whatwg/sg and making it suitable for whatwg.org"
```

Comment:
Seems like this should happen either way (outside the if/else) so that we can test the conversion when people submit changes to the script.

Reply:
The correct invocation is `pip install markdown py-gfm`

Reply:
@domenic how would that actually test the conversion though? Or do you mean locally? Locally seems reasonable, but I'd like to avoid a git clone for each PR.